store/index.js

```diff
@@ -146,7 +146,7 @@ export const getters = {
 
     getSelectedCategoricalOption: (p_state) => (p_column, p_rawValue) => {
 
-        return p_state.dataDictionary.annotated[p_column]?.valueMap[p_rawValue] ?? "";
```

Comment:
I'm curious, what was the behavior of the bug? Would the expression not short circuit for every null/undefined chain option (except the first one) and instead throw an error if a property didn't exist?

Reply:
It's a good question especially if we want to use these operators in the future. It was looking for `p_rawValue` and stating that it could not find the string in `p_rawValue` in `valueMap` – which is confusing because we expect the nullish coalescing operator (`??`) to resolve anything `undefined` and return with value on the right (empty string).

Old version:
`return p_state.dataDictionary.annotated[p_column]?.valueMap[p_rawValue] ?? "";`

Here it assumes that `p_column` will be found in `annotated` and `p_rawValue` will be found in `valueMap`.

New version:
`return p_state.dataDictionary.annotated?.[p_column]?.valueMap?.[p_rawValue] ?? "";`

We need to add the `?.` operator before the array index syntax to get it to resolve appropriately.

(The extra `?.` in front of `[p_column]` is done here out of caution. In fact, it could be removed. We should technically never get here without the string in `p_column` as a key in `annotated` – but if we did it would resolved as `undefined` appropriately and hit the nullish coalescing operator and return the empty string.)